vivisect/vdbext.py

```diff
@@ -99,9 +99,9 @@ def fireattach(trace, pid):
 
 
 @idlethread
-def runVdb(vivgui, pid=None):
+def runVdb(vivgui, pid=None, trace=None):
```

Comment:
So you added the param, but I don't see anybody using it? Is this just future proofing?

Reply:
yes... ish.  it was originally part of the `VdbCli.do_gui()` fix, but it felt right to leave it when i chose the more explicit method.